ui/docker/Dockerfile-ui

```diff
@@ -21,6 +21,7 @@ WORKDIR /app
 
 COPY docker/conf.d /etc/nginx/conf.d
 
+ENV REACT_APP_ENABLE_LOCAL_LOGIN YES
```

Comment:
Is this variable required during build? If not isn't better to set it in docker-compose.yml file?

Reply:
yes we need this during build